localization/english/wc_holy_orders_l_english.yml

```diff
@@ -2,6 +2,246 @@
  #########################
  #		Holy Orders		     #
  #########################
+ ### Any Elemental Group ###
+ holy_order_elemental_article:0 "the "
+ holy_order_elemental:0 "Elementals of [TITLE.GetNameNoTierNoTooltip]"
+ ### Air Group ###
+ holy_order_conclave_of_wind_article:0 "the "
+ holy_order_conclave_of_wind:0 "Conclave of Wind"
+ holy_order_cloudriders_article:0 "the "
+ holy_order_cloudriders:0 "Cloudriders of [FAITH.HighGodName]"
+ holy_order_skywarriors_article:0 "the "
+ holy_order_skywarriors:0 "Skywarriors of [FAITH.WarGodName]"
+ holy_order_eye_of_the_storm_article:0 "the "
+ holy_order_eye_of_the_storm:0 "Eye of the Storm"
+ ### Earth Group ###
+ holy_order_rock_hard_guard_article:0 "the "
+ holy_order_rock_hard_guard:0 "Rock Hard Guard"
+ holy_order_earthprotectors_article:0 "the "
+ holy_order_earthprotectors:0 "Earthprotectors of [FAITH.HighGodName]"
+ holy_order_immovable_mountains_article:0 "the "
+ holy_order_immovable_mountains:0 "Immovable Mountains"
+ holy_order_epicentre_of_destruction_article:0 "the "
+ holy_order_epicentre_of_destruction:0 "Epicentre of Destruction"
+ ### Fire Group ###
+ holy_order_volcanic_activity_article:0 "the "
+ holy_order_volcanic_activity:0 "Volcanic Activity"
+ holy_order_fireservants_article:0 "the "
+ holy_order_fireservants:0 "Fireservants of [FAITH.HighGodName]"
+ holy_order_hot_to_the_touch_article:0 "the "
+ holy_order_hot_to_the_touch:0 "Hot to the Touch"
+ ### Water Group ###
+ holy_order_hydraxian_waterlords_article:0 "the "
+ holy_order_hydraxian_waterlords:0 "Hydraxian Waterlords"
+ holy_order_low_tide_article:0 "the "
+ holy_order_low_tide:0 "Low Tide"
+ holy_order_high_tide_article:0 "the "
+ holy_order_high_tide:0 "High Tide"
+ holy_order_highest_wave_article:0 "the "
+ holy_order_highest_wave:0 "Highest Wave"
+ holy_order_seahunters_article:0 "the "
+ holy_order_seahunters:0 "Seahunters of [FAITH.HighGodName]"
+
+ ### Ape Group ###
+ holy_order_peaceful_ways_article:0 "the "
+ holy_order_peaceful_ways:0 "Peaceful Ways"
+ holy_order_controlled_savagery_article:0 "the "
+ holy_order_controlled_savagery:0 "Controlled Savagery"
+ holy_order_peacekeepers_article:0 "the "
+ holy_order_peacekeepers:0 "Peacekeepers of [FAITH.HighGodName]"
+
+ ### Arcane Group ###
+ holy_order_masters_of_essence_of_magic_article:0 "the "
+ holy_order_masters_of_essence_of_magic:0 "Masters of the Essence of Magic"
+ holy_order_researchers_article:0 "the "
+ holy_order_researchers:0 "Researchers of [FAITH.HighGodName]"
```

Comment:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126021488-0011ee8a-1116-47ba-b8d1-65fd8f9b9d94.png)

</details>

The suggestion will make the ancestors uppercase but you may reword it since I guess you supposed high god to be Arcane?

```suggestion
 holy_order_researchers:0 "Researchers of [FAITH.HighGodName|U]"
```